crates/bevy_render/src/mesh/mesh.rs

```diff
@@ -453,20 +400,117 @@ pub mod shape {
             }
         }
     }
+
+    /// A Square prism. Just like a cube, but you can specify the dimensions.
+    pub struct SquarePrism {
+        pub min_x: f32,
+        pub max_x: f32,
+
+        pub min_y: f32,
+        pub max_y: f32,
+
+        pub min_z: f32,
+        pub max_z: f32,
+    }
+
+    impl Default for SquarePrism {
+        fn default() -> Self {
+            SquarePrism {
+                min_x: 0.0,
```

Comment:
In interest of keeping the translation around local (0, 0), should this be just `length`, `width`, and `height`, and not allow below 0?

 I'm actually slightly surprised the `cube` shape isn't centered around (0, 0), dividing the size by 2 for the negative and positive, I'm not sure if that's intentional or not... but obviously that's nothing you changed

Reply:
I'll change it, it's probably better if the default is centred on 0,0,0. I will also change the shape, it should be a rectangular prism by default, not a cube.